.circleci/config.yml

```
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
+          command: git remote set-url origin https://${GITHUB_TOKEN}@github.com/VirtusLab/git-machete-intellij-plugin.git
+      - run:
+          name: Configure github user information
+          command: |
+            git config user.email "gitmachete@virtuslab.com"
+            git config user.name "Git Machete Bot"
+      - run:
+          # language=sh
+          command: |
+            ./gradlew updateEapBuildNumber
+            if ! git diff --exit-code --quiet intellijVersions.properties ; then
```

Nit: even more elegant would be sth like

```suggestion
            if ./gradlew updateEapBuildNumber --exit-code; then
```

Ofc the support for `--exit-code` option (exit with 1 if nothing updated, 0 if something updated) would need to be added to the task... see the `ResolveIntellijVersions` task you've removed recently, it's got some command-line option implemented

---

oki :> I could not find any elegant solution to return exit code from gradle task. The only way I found is to return (if exit code should be 0) and throw some Exception (if exit code should be 1 ?